openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -142,25 +154,46 @@ def handle_notify_credentials(options, course_keys):
             notify_programs=options['notify_programs']
         )
 
-    logger.info('notify_credentials finished')
-
 
 def send_notifications(
-    certs, grades, site_config=None, delay=0, page_size=100, verbose=False, notify_programs=False
+    certs,
+    grades,
+    site_config=None,
+    delay=0,
+    page_size=100,
+    verbose=False,
+    notify_programs=False,
+    revoke_program_certs=False
```

Comment:
`revoke_program_certs` is a new keyword-arg added that will eventually be passed in from the management command/jenkins job. 

This is what will ultimately decide whether the system should check to see if a program cert should be revoked from a learner.